sdk/appconfiguration/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/manager/models/FeatureDefinition.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.manager.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.validation.annotation.Validated;
+
+import com.azure.spring.cloud.feature.manager.implementation.models.DynamicFeature;
+
+/**
+ * The definition of a dynamic feature.
+ */
+@Validated
+public class FeatureDefinition {
+
+	private String name;
+
+	private String assigner;
+
+	private List<FeatureVariant> variants;
+
+	/**
+	 * Definition of a Dynamic Feature.
+	 * 
+	 * @param feature        name of the feature
+	 * @param dynamicFeature dynamic feature object to be made into a definition
+	 */
+	public FeatureDefinition(String feature, DynamicFeature dynamicFeature) {
+		this.name = feature;
+		this.assigner = dynamicFeature.getAssigner();
+		this.variants = new ArrayList<FeatureVariant>();
+
+		for (int i = 0; i < dynamicFeature.getVariants().size(); i++) {
+			variants.add(dynamicFeature.getVariants().get(String.valueOf(i)));
+		}
```

Comment:
nit: using stream will be much shorter

Reply:
I think the issue I found with making this a stream is that I need the order they are in the list to be the same. Wouldn't be guaranteed with a stream.